DEVELOPING.md

```diff
@@ -15,7 +15,7 @@ You would only do this once after you cloned the repository.
 1.  `yarn` to bring in all the top-level dependencies.
 1.  Open the project in your editor of choice.
 
-When you are ready to commit
+###### When you are ready to commit
```

Comment:
Is that supposed to be 6 of 'em?

Reply:
I suppose that's about equivalent to a dash while converting it to a heading.